requirements.txt

```diff
@@ -7,13 +7,12 @@ shapely>=1.6
 geopandas>=0.7
 dask[complete]>=2.6
 pyproj>=2.5
-requests>=2.0
-roocs-utils>=0.3.0
-cf-xarray>=0.5.1
-scipy>=1.5.4
-# roocs-utils @ git+https://github.com/roocs/roocs-utils.git@master#egg=roocs-utils
+cf-xarray>=0.6.3
+#cf-xarray @ git+https://github.com/xarray-contrib/cf-xarray/@main#egg=cf-xarray
 roocs-grids @ git+https://github.com/roocs/roocs-grids.git@main#egg=roocs-grids
-
 # Optional (if using regridding)
-# pangeo-xesmf>=0.6.0
```

Comment:
It provides the same functionality, it had to be named pangeo-xesmf since xesmf was already taken by the original xesmf repo. That might change in the future.

I think it is optional on purpose, so clisops is installed by default without regridding functionalities and that case is tested. We can discuss this next week in the 34e-Ouranos-CRIM meeting and activate the installation if noone objects.

Reply:
pip cannot install esmpy which seems to be why this is deactivated.